test/byContract/IexecMaintenance/IexecMaintenance.test.ts

```diff
@@ -112,7 +112,33 @@ describe('Maintenance', async () => {
     });
 
     describe('Import score', () => {
-        // Not tested
+        it('Should import score', async () => {
```

Comment:
```suggestion
        it('[TODO] Should import score', async () => {
```
So that we can see it in test list or with ctrl + f